dtmi/terasic/fcc/de10_nano_threshold-1.json

```diff
@@ -0,0 +1,219 @@
+        "en": "Accelerometer Z Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Accelerometer Z axis"
+      },
+      "name": "az",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:GX;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Gyroscope X Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Gyroscope X axis"
+      },
+      "name": "gx",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:GY;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Gyroscope Y Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Gyroscope Y axis"
+      },
+      "name": "gy",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:GZ;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Gyroscope Z Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Gyroscope Z axis"
+      },
+      "name": "gz",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:MX;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Magnetometer X Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Magnetometer X axis"
+      },
+      "name": "mx",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:MY;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Magnetometer Y Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Magnetometer Y axis"
+      },
+      "name": "my",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:MZ;1",
+      "@type": "Property",
+      "displayName": {
+        "en": "Magnetometer Z Axis"
+      },
+      "description" : {
+        "en" : "Max and Min values of MPU9250 Magnetometer Z axis"
+      },
+      "name": "mz",
+      "schema": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "writable": true
+    }
+  ],
+  "schemas": [
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "displayName": {
+            "en": "Minimum Value"
+          },
+          "description" : {
+            "en" : "Minimum sensor value in float to ?????"
```

Comment:
Just want to confirm that the `?????` value for a few of the description.en properties  in the model are intended?